src/sentry/event_manager.py

```diff
@@ -167,6 +167,7 @@
 DEFAULT_GROUPHASH_IGNORE_LIMIT = 3
 GROUPHASH_IGNORE_LIMIT_MAP = {
     GroupType.PERFORMANCE_N_PLUS_ONE_DB_QUERIES: 3,
+    GroupType.PERFORMANCE_N_PLUS_ONE_API_CALLS: 100,
```

Comment:
I think 100 as the limit for this issue type might be a bit too high, no?

Reply:
🤔 yeah, you're right, especially for LA/EA. I'll stick with the default for LA/EA. Maybe it can be even lower?

Reply:
No, I initially set it to 100 because I was following the lead of the slow span detector, which is set to 100, that's my mistake!